drizzlepac/hlautils/astroquery_utils.py

```diff
@@ -93,14 +93,11 @@ def retrieve_observation(obsid, suffix=['FLC'], archive=False, clobber=False):
 
     if not clobber:
         for rownum in rows_to_remove[::-1]:
-            if not manifest:
-                return all_images
-            else:
+            if manifest:
```

Comment:
@dulude The idea behind https://github.com/spacetelescope/drizzlepac/pull/307#issuecomment-481776421 was that `manifest` cannot become empty in this look and therefore `if manifest` can be pulled out of the loop.

Reply:
Resolved issue in commit [c88346f](https://github.com/spacetelescope/drizzlepac/pull/307/commits/c88346f91194cfe65ad9b46389ca83a94217dd59)